# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/audioconduit_unittests.cpp
# Commit: 2a373d864092
# Full Hash: 2a373d864092cdd0eb970b247e27f320650ec2c7
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 9
# Description:
#   Bug 1654112 - Rename MediaSessionConduit::DeleteStreams to Shutdown, as VideoConduit does more than just delete streams. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112856
# ==============================================================================

diff -r 09db50c8f370 -r 2a373d864092 media/webrtc/signaling/gtest/audioconduit_unittests.cpp
--- a/media/webrtc/signaling/gtest/audioconduit_unittests.cpp	Wed Apr 14 17:36:40 2021 +0200
+++ b/media/webrtc/signaling/gtest/audioconduit_unittests.cpp	Wed Apr 14 17:40:24 2021 +0200
@@ -26,7 +26,7 @@
             mCallWrapper, GetCurrentSerialEventTarget())) {}
 
   ~AudioConduitTest() override {
-    mAudioConduit->DeleteStreams();
+    mAudioConduit->Shutdown();
     mCallWrapper->Destroy();
   }
 